packages/office-ui-fabric-react/package.json

```diff
@@ -79,6 +79,7 @@
     "@uifabric/set-version": "^7.0.5",
     "@uifabric/styling": "^7.10.8",
     "@uifabric/utilities": "^7.12.0",
+    "@fluentui/react-focus": "^0.1.0",
```

Comment:
Can we have this in a separate PR first and make sure nobody busts?

Reply:
done